.changes/unreleased/Fixes-20230508-060926.yaml

```diff
@@ -0,0 +1,6 @@
+kind: Fixes
```

Comment:
Is this a fix or a feature? In other words, did we expect it to be at the cursor level and we just didn't do that? Or did users expect it to be at the cursor level and we're meeting their needs now?

Reply:
Ah, makes sense.